opportunity.py

```diff
@@ -0,0 +1,92 @@
+from dataclasses import dataclass
+from dotenv import load_dotenv
+import psycopg2
+from typing import List
+import os
+import utility as utils 
+
+load_dotenv()
+
+table_name = os.getenv("DB_TABLE")
+
+@dataclass
+class Opportunity:
+    """Class for ..."""
+    # TODO
+
+
+def list_all_opportunities() -> List[object]:  # Return type of opportunity
```

Comment:
Return a list of the `Opportunity` dataclass.
```suggestion
def list_all_opportunities() -> List[Opportunity]:  # Return type of opportunity
```